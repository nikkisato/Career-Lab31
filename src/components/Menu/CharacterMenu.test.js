import React from 'react';
import { shallow } from 'enzyme';
import CharacterMenu from './CharacterMenu';

describe('CharacterMenu component', () => {
  it('renders CharacterMenu', () => {
    const wrapper = shallow(<CharacterMenu />);
    expect(wrapper).toMatchSnapshot();
  });
});
