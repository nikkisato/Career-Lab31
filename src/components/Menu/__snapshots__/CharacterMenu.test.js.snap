// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterMenu component renders CharacterMenu 1`] = `
<Fragment>
  <select>
    <Selection
      options={
        Array [
          Object {
            "name": "Bender",
            "value": "bender",
          },
          Object {
            "name": "Fry",
            "value": "fry",
          },
          Object {
            "name": "Leela",
            "value": "leela",
          },
          Object {
            "name": "Professor Farnsworth",
            "value": "professor-farnsworth",
          },
          Object {
            "name": "Amy",
            "value": "amy",
          },
          Object {
            "name": "Zapp-Brannigan",
            "value": "zapp-brannigan",
          },
          Object {
            "name": "Lurr",
            "value": "lurr",
          },
          Object {
            "name": "Dr Zoidberg",
            "value": "dr-zoidberg",
          },
          Object {
            "name": "Bob Barker",
            "value": "bob-barker",
          },
          Object {
            "name": "Hermes",
            "value": "hermes",
          },
          Object {
            "name": "Morgan Proctor",
            "value": "morgan-proctor",
          },
          Object {
            "name": "Mom",
            "value": "mom",
          },
          Object {
            "name": "Robot-Mob",
            "value": "robot-mob",
          },
          Object {
            "name": "Giant Bender",
            "value": "giant-bender",
          },
          Object {
            "name": "Kif",
            "value": "kif",
          },
        ]
      }
    />
  </select>
  <button>
    Get Quote
  </button>
</Fragment>
`;
